public/js/maps.js

```diff
@@ -22,14 +22,13 @@ function initMap() {
     ],
   });
   bounds = new google.maps.LatLngBounds(HOUSTON_COORDS);
-  generateMarkers(organizations);
 }
 
 /**
  * Add markers to the map based on the data in the data array passed in.
```

Comment:
nit: Adds ... 

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments